src/store/store.js

```diff
@@ -0,0 +1,30 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+// import authServices from '@/api/authService';
+import createPersistedState from "vuex-persistedstate";
+
+Vue.use(Vuex)
+
+
+export default new Vuex.Store({
+    state: {
+        user: {
+            loggedIn: false,
+            token: '',
+        }
+    },
+
+    getters: {
+        userState(state) {
+            return state.user;
+        }
+    },
+
+    mutations: {
+        setUserState(state, token = '') {
+            state.user.loggedIn = !!token;
+            state.user.token = token || '';
+        },
+    },
+    plugins: [createPersistedState()],
```

Comment:
FYI if we split the store into modules you can pass on objet to this function and specify the paths
```
plugins: [
    createPersistedState({
      paths: ['auth']
    })
  ],
```
the object is the options